### ISSUE TITLE
Game crashes in DrawTextbox_895D0 when displaying text "Destroy this for its mana" in 320x200

### ISSUE THREAD

**Issue by thobbsinteractive** [thobbsinteractive/magic-carpet-2-hd#175]

**Comment by thobbsinteractive** [comment]
So I looked at the method. At first the problem seemed to be that the TextBox width when divided by the character size was less than the number of characters due to rounding. So I fixed this with the ceil() method:
`int width = std::ceil((float)textbox->textBoxWidth_0x4 / (float)textbox->charWidth_0x10);`

However that only got me a bit further before it crashed. The Text boxes looked very small, a lot of wrapping was happening. 
E.g:
![image](https://user-images.githubusercontent.com/19335188/187050024-c2d9714a-4d7c-429a-b935-253567bb5ed1.png)
So I think the NEXT crash was due to the textbox size being too small for the number of text characters:
For the Message: "Use possession spell to possess mana" it is trying to fit it into a text box of width "8"
I think the word "possession" is too long here
